config/routes.rb

```diff
@@ -1,5 +1,6 @@
 MaycampArena::Application.routes.draw do
 
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
```

Comment:
Please use JSON style hashes (`omniauth_callbacks: ...`). This is the convention we try to keep around the project (when writing new code and refactoring old one).